cache/src/main/antlr4/net/runelite/cache/script/assembler/rs2asm.g4

```diff
@@ -24,7 +24,7 @@
  */
 grammar rs2asm;
 
-prog: NEWLINE* (header NEWLINE+)* (line NEWLINE+)+ ;
+prog: NEWLINE* (header NEWLINE+)* (line NEWLINE*)+ EOF;
```

Comment:
I probably don't want newlines optional on this.

Reply:
I did not found better way how to make the last newline optional what would not freeze the assembler. Any tips?